### PR TITLE
Make sqlippool generic rather than protocol specific

### PR DESCRIPTION
**Pull request by ndptech**
Rename the queries used by sqlippool to reflect their actions rather than protocol specific packets

### CODE REVIEW

File: raddb/mods-config/sql/ippool/mssql/queries.conf [hunk 1]
```diff
@@ -112,13 +112,13 @@ pool_check = "\
 #allocate_commit = ""
 
 #
-#  This series of queries frees an IP number when an accounting START record arrives.
+#  This series of queries frees an IP number when an accounting START / ALIVE record arrives.
```
Comment (alandekok) [inline raddb/mods-config/sql/ippool/mssql/queries.conf:115]: Maybe not "frees" an IP address?  But extends it?

File: src/modules/rlm_sqlippool/rlm_sqlippool.c [hunk 2]
```diff
@@ -125,32 +125,32 @@ static CONF_PARSER module_config[] = {
 	{ FR_CONF_OFFSET("pool_check", FR_TYPE_STRING | FR_TYPE_XLAT, rlm_sqlippool_t, pool_check), .dflt = "" },
 
 
-	{ FR_CONF_OFFSET("extend_begin", FR_TYPE_STRING | FR_TYPE_XLAT, rlm_sqlippool_t, extend_begin), .dflt = "START TRANSACTION" },
+	{ FR_CONF_OFFSET("extend_begin", FR_TYPE_STRING | FR_TYPE_XLAT, rlm_sqlippool_t, extend_begin), .dflt = "" },
```
Comment (alandekok) [inline src/modules/rlm_sqlippool/rlm_sqlippool.c:128]: TBH you could just remove the `.dflt = ""`, and it would be fine.

**Comment by alandekok** [comment]
Can you rebase these on top of the "master" branch, instead of merging it?  That way the history looks cleaner.

But overall this is very good, thanks.

**Comment by arr2036** [comment]
@ndptech  can you write up a series of generic method names for ippool modules that we can discuss and standardise on.  I don't believe we currently support overrides based on section names, but we should support overrides for generic methods like `sqlippool.update`.  We should add these methods as well as section bindings to both ippool modules.

Maybe `alloc`, `update`, `release`, `bulk-release`, `query`, `bulk-query`.

**Pull request merged**